docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -604,7 +606,8 @@ For example, you have the following configuration:
 %test.quarkus.oidc.credentials.secret=secret
 ----
 
-To start, add the same dependency, `quarkus-test-oidc-server`, as described in the <<integration-testing-wiremock>> section.
+ifndef::no-deprecated-test-resource[]
```

Comment:
OidcTestClient is a new option which can be used to test against live remote endpoints

Reply:
Thanks, Sergey. These conditionals were added in https://github.com/quarkusio/quarkus/pull/41712. I am proposing to remove them in https://github.com/quarkusio/quarkus/pull/41977. Does that address this issue?